config/default.yml

```diff
@@ -1,6 +1,120 @@
 # Write it!
 
+AllCops:
+  EnabledByDefault: true
+  Exclude:
+    - 'bin/**/*'
+    - 'db/schema.rb'
+    - 'vendor/**/*'
+
+Bundler/OrderedGems:
+  TreatCommentsAsGroupSeparators: false
+
 Fairmoney/NoIfInTestDescriptions:
   Description: 'Test descriptions should not include "if", use "when" instead'
   Enabled: true
   VersionAdded: '0.91'
+
+Layout/CaseIndentation:
+  EnforcedStyle: end
+
+Layout/EndAlignment:
+  EnforcedStyleAlignWith: variable
+
+Layout/LineLength:
+  Max: 149
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+  IndentationWidth: 2
+
+Layout/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+Metrics/BlockLength:
+  Exclude:
+    - 'config/environments/*.rb'
+    - 'config/routes.rb'
+    - 'db/migrate/*'
+    - 'spec/**/*'
+
+Lint/ConstantResolution:
+  Enabled: false
+
+Metrics/MethodLength:
+  Max: 100
+
+Naming/VariableNumber:
+  EnforcedStyle: snake_case
+
+RSpec/DescribedClass:
+  EnforcedStyle: explicit
+
+RSpec/ExampleLength:
+  Max: 100
+
+RSpec/MessageExpectation:
+  Enabled: false
+
+RSpec/MessageSpies:
+  EnforcedStyle: receive
+
+RSpec/MultipleExpectations:
+  Max: 30
```

Comment:
30 expects in a single test ? :open_mouth: :open_mouth: :open_mouth: :open_mouth: :open_mouth:

Reply:
Default is 1 expect :-D will let each project handle it

Reply:
Ideally you only expect your test description to be true

In practice I would see something like max 3 expects for a unit test and max 7-10 expects for an integration test